src/remix/sdk-setup.ts

```diff
@@ -0,0 +1,499 @@
+import type { ExportNamedDeclaration, Program } from '@babel/types';
+
+// @ts-ignore - magicast is ESM and TS complains about that. It works though
+import type { ProxifiedModule } from 'magicast';
+
+import * as fs from 'fs';
+import * as path from 'path';
+import * as url from 'url';
+import chalk from 'chalk';
+import { parse } from 'semver';
+
+// @ts-ignore - clack is ESM and TS complains about that. It works though
+import clack from '@clack/prompts';
+
+// @ts-ignore - magicast is ESM and TS complains about that. It works though
+import { builders, generateCode, loadFile, writeFile } from 'magicast';
+import * as recast from 'recast';
+import {
+  ERROR_BOUNDARY_TEMPLATE_V2,
+  HANDLE_ERROR_TEMPLATE_V2,
+} from './templates';
+import { PackageDotJson, getPackageVersion } from '../utils/package-json';
+
+export type PartialRemixConfig = {
+  unstable_dev?: boolean;
+  future?: {
+    v2_dev?: boolean;
+    v2_errorBoundary?: boolean;
+    v2_headers?: boolean;
+    v2_meta?: boolean;
+    v2_normalizeFormMethod?: boolean;
+    v2_routeConvention?: boolean;
+  };
+};
+
+const REMIX_CONFIG_FILE = 'remix.config.js';
+
+// Copied from sveltekit wizard
+function hasSentryContent(fileName: string, fileContent: string): boolean {
+  if (fileContent.includes('@sentry/remix')) {
+    clack.log.warn(
+      `File ${chalk.cyan(path.basename(fileName))} already contains Sentry code.
+Skipping adding Sentry functionality to ${chalk.cyan(
+        path.basename(fileName),
+      )}.`,
+    );
+
+    return true;
+  }
+  return false;
+}
+
+/**
+ * Copied from sveltekit wizard
+ * We want to insert the init call on top of the file but after all import statements
+ */
+function getInitCallInsertionIndex(originalHooksModAST: Program): number {
```

Comment:
If this is from the sveltekit wizard, @Lms24 do you think it's appropriate if we extract into a util?